backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -317,52 +326,75 @@ export class AdminResolver {
     @Arg('id', () => Int) id: number,
     @Ctx() context: Context,
   ): Promise<boolean> {
-    const pendingCreation = await AdminPendingCreation.findOne(id)
-    if (!pendingCreation) {
+    const contribution = await Contribution.findOne(id)
+    if (!contribution) {
```

Comment:
`confirmPendingCreation` should be ranemd as well!
Eventhough `confirmPendingCreation`.